app/views/pages/teacher_new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <h1>Tick to make teacher</h1>
+  <% @students.each do|student|%>
+    <%= student.first_name %>
```

Comment:
we should probably add <%= student.last_name %> in case we have two students with the same first name (e.g. Laura) to make sure which one to choose when making the student a teacher

Reply:
yes this view is still very raw, i am thinking of displaying all students and teachers and then have a tickmark if u want to change a student to a teacher